src/en/theblank/src/eu/kanade/tachiyomi/extension/en/theblank/TheBlank.kt

```diff
@@ -1,109 +1,488 @@
+            title = data.title
+            thumbnail_url = data.image?.let { baseUrl + it }
+            author = data.author
+            artist = data.artist
+            description = buildString {
+                data.description?.also {
+                    append(it.trim(), "\n\n")
                 }
-                startActivity(intent)
-                false
+                data.releaseYear?.also {
+                    append("Release: ", it, "\n\n")
+                }
+                data.alternativeName?.also {
+                    append("Alternative name: ", it)
+                }
+            }.trim()
+            genre = buildList {
+                data.type?.name?.also(::add)
+                data.genres.mapTo(this) { it.name }
+            }.joinToString()
+            status = when (data.status) {
+                "ongoing", "upcoming" -> SManga.ONGOING
+                "finished" -> SManga.COMPLETED
+                "dropped" -> SManga.CANCELLED
+                "onhold" -> SManga.ON_HIATUS
+                else -> SManga.UNKNOWN
             }
-            screen.addPreference(this)
         }
     }
-}
 
-private fun startActivity(intent: Intent) {
-    intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TOP)
-    val app = Injekt.get<Application>()
-    try {
-        app.startActivity(intent)
-    } catch (e: Throwable) {
-        Log.e("TheBlank", "failed to start activity", e)
-        Toast.makeText(app, e.message, Toast.LENGTH_LONG).show()
+    private fun createThumbnailUrl(imagePath: String?): String? {
+        return baseHttpUrl.newBuilder()
+            .encodedPath(imagePath ?: return null)
+            .fragment(THUMBNAIL_FRAGMENT)
+            .toString()
+    }
+
+    override fun chapterListRequest(manga: SManga) =
+        mangaDetailsRequest(manga)
+
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val data = response.parseAs<MangaResponse>().props.serie
+        val hidePremium = preferences.getBoolean(HIDE_PREMIUM_PREF, false)
+
+        return data.chapters
+            .filter { !(it.isPremium && hidePremium) }
+            .map {
+                SChapter.create().apply {
+                    url = baseUrl.toHttpUrl().newBuilder().apply {
+                        addPathSegment("serie")
+                        addPathSegment(data.slug)
+                        addPathSegment("chapter")
+                        addPathSegment(it.slug)
+                    }.build().encodedPath
+                    name = buildString {
+                        if (it.isPremium) {
+                            append("\uD83D\uDD12 ") // lock emoji
+                        }
+                        append(it.title)
+                    }
+                    date_upload = dateFormat.tryParse(it.createdAt)
+                }
+            }.asReversed()
+    }
+
+    private val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'", Locale.ENGLISH)
```

Comment:
Not using any locale specific patterns, use:

```suggestion
    private val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSSSS'Z'", Locale.ROOT)
```